packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -460,9 +460,9 @@ test('first', async t => {
     'unused collateral remains after liquidation',
   );
 
-  t.deepEqual(await E(vaultFactory).getPenaltyAllocation(), {
-    RUN: AmountMath.make(runBrand, 30n),
-  });
+  // ??? remove penalty pool from vault factory
+  // used to be funded from liquidation penalty
+  t.deepEqual(await E(vaultFactory).getPenaltyAllocation(), {});
```

Comment:
I think it's correct to take the penalty funds out of the vault, but then they have to be accessible (even if we aren't yet retrieving them) from the liquidation contracts.

Reply:
Yes, but I'm not sure that `test-vaultFactory` needs to be concerned with that. It's a concern of the liquidation contracts what they do with the `liquidationPenalty` argument.

Reply:
I see I misread your comment. Agree completely and working on it.